hub/config/src/test/java/uk/gov/ida/hub/config/data/ManagedEntityConfigRepositoryTest.java

```diff
@@ -110,5 +110,36 @@ public void getReturnsOverriddenConfigWhenRemoteFoundAndEnabled() {
         assertThat(result.get().getEncryptionCertificate().getBase64Encoded().get()).isEqualTo(REMOTE_CERT);
         assertThat(result.get().getEncryptionCertificate().getCertificateOrigin()).isEqualTo(CertificateOrigin.SELFSERVICE);
     }
+    
+    @Test(expected = NoCertificateFoundException.class)
```

Comment:
I'm confused by this test - if we're expecting the `get` to throw then why do we have the asserts? If `get` throws the test will never get that far down? 'd prefer to use `assertThatThrownBy(() ​-> ...)` instead of `(expected = ...)` so it's clear which line we're expecting to throw